test/Preprocessing/pp001.F

```diff
@@ -1,3 +1,4 @@
+!RUN: %flang -E %s
```

Comment:
These files were tests before. I'm not sure this adds much because there is no checking of results.

Reply:
They don't need to be checked in this case.

Reply:
Sorry, I meant to write "these files were **not** tests before". So running them as if they are tests is a change. I'm wondering if there is any benefit to doing that.

Reply:
I suppose not, but we thought to port them unless there is an objection since their addition implies they will be included as a test in future.

Reply:
The third option is to leave them as they were, just examples that are not executed as tests.

If you think there is value in executing them to check there are no compilation errors, that's fine. But that should be made explicit in the PR (or a separate PR) saying that's what you're doing that and why. Don't bury it under all of the changes related to porting tests.

Reply:
Sorry for not being more clear in the description.

Lit will emit an error for test files in the test directory (identified by file extension) that contain no runlines. So to port these tests to lit, we have three options:
 1. Add RUN lines to them (our chosen option)
 2. Explicitly exclude this directory from lit's searching process
 3. add a lit.local.cfg in this test directory that changes the file extensions associated with tests.
 4. Rename the file extensions (yuk!)

If we don't like any of these options we could leave them behind in the unittests directory with no change and revisit later or we could delete them altogether. 

To me, it seems pointless and unhelpful to have test files in the test directory that look like tests but are intentionally not executed. If we don't want to execute these then I think we should delete them. Strong view weakly held though. What would you be happy with?

Reply:
1 and 2 are both fine with me, as is moving them to a different directory, say `examples`. I just want it to be an explicit decision that is made, not something that happens accidentally.

@klausler, what do you think we should do with the preprocessor tests?